jwst/resample/resample.py

```diff
@@ -164,8 +163,7 @@ def do_drizzle(self):
                 inwht = resample_utils.build_driz_weight(img,
                     weight_type=self.weight_type,
                     good_bits=self.good_bits)
-                driz.add_image(img.data, img.meta.wcs, inwht=inwht,
-                    expin=img.meta.exposure.exposure_time)
```

Comment:
So given that exptime is still an allowed choice for weighting (just no longer the default), what happens when someone selects exptime but the exposure_time is no longer being passed to the lower-level functions?

Reply:
Ah, good catch.  Let me look at my unit tests and see what I missed.

That raises the larger question, should we even allow exposure time to be used?  It clearly gives the _wrong_ answer for multi-read up-the-ramp data, so maybe it should just be completely removed?

Reply:
So what's the verdict here? Does it still work when selecting "exptime" weighting?